### ISSUE TITLE
Inconsistent rendering priority with School Grounds

### ISSUE THREAD

**Issue by WinterB1860** [gravitystorm/openstreetmap-carto#3348]
If you have an Area object with the tag of amenity=school and another area object inside tagged as landuse=grass, it will render the grass object on top.
If you use a multipolygon to determine where the grass is, the grass will not render

If you have a multipolygon with the tag of amenity=school, both area and multipolygon grass objects render on top.

**Comment by kocio-pl** [comment]
Could you give examples with URLs and screenshots, please?

**Comment by WinterB1860** [comment]
Hm, I can't seem to reproduce the issue other than using a large relationship of local grass.

I was able to resolve what issues I was seeing where it wasn't rendering the multipolygon grass if it passed completely through the schoolgrounds, unless I had a way tagged wrong to properly enclose the schoolgrounds

https://i.imgur.com/yZgAFdv.png
https://www.openstreetmap.org/#map=17/44.63436/-68.85546

In my screenshot there are two initial groups of Grass (1 and 2)
The red lines are the school limits, beyond what limits are set by the woods
The purple line in the left side is where I initially had an extra part of a way with school grounds relationship.

Initially the school on the lower right wasn't rendering grass when it was a multipolygon no matter what, while the upper left school rendered it. When I fixed the errant line (purple) it seemed to stop rendering the multipolygon grass

Due to the issues, I tested in another area and small sections of objects and multipolygons render properly with grass over the school grounds

In order to make the map render properly what I ended up doing is to separate the multipolygons by making a new grass relationship for each school grounds, splitting up the grass over multiple multipolygons in the area. This seemed to make sure it is being rendered properly.
